### PR TITLE
Update OpenShift docs

### PR DESCRIPTION
**Pull request by bianchi2**
This PR introduces a few updates to OpenShift docs, namely:

* turning off securityContext (OpenShift will do it on its own)
* add docs on how to solve permission denied errors in app installation directory (declare those as emptyDir)

### CODE REVIEW

File: docs/docs/platforms/OPENSHIFT.md [hunk 1]
```diff
@@ -26,11 +26,28 @@ It depends on the storage backend though.
...
+## Permission issues
+
+If a container start without anyuid enabled, applications can't write to `${APPLICATIOn_HOME}/logs`, `${APPLICATIOn_HOME}/work` and `${APPLICATIOn_HOME}/temp`.
+If you see in logs that the server fails to start with `permission denied` errors, you may want to declare these directories as runtime volumes. To do so, you need to declare additional volume mounts and additional volumes in values.yaml:
+
+```
+confluence:
+  additionalVolumeMounts:
+    - name: tomcat-work
+      # this example if for Confluence
```
Comment (nghazali) [inline docs/docs/platforms/OPENSHIFT.md:42]:
```suggestion
      # this example is for Confluence
```

File: docs/docs/platforms/OPENSHIFT.md [hunk 2]
```diff
@@ -26,11 +26,28 @@ It depends on the storage backend though.
...
 ## Set no security context
 
-As an alternative, (if letting containers run as pre-defined users is not possible), set `product_name.securityContext` to `{}`.
+As an alternative, (if letting containers run as pre-defined users is not possible), set `product_name.securityContextEnabled` to `false`, for example, `confluence.securityContextEnabled: false`
 As a result the container will start as a user with an OpenShift generated ID.
 Typically, NFS permission fixer job isn't required when no security context is set.
 
+## Permission issues
+
+If a container start without anyuid enabled, applications can't write to `${APPLICATIOn_HOME}/logs`, `${APPLICATIOn_HOME}/work` and `${APPLICATIOn_HOME}/temp`.
```
Comment (nanux) [inline docs/docs/platforms/OPENSHIFT.md:35]:
```suggestion
If a container start without anyuid enabled, applications can't write to `${APPLICATION_HOME}/logs`, `${APPLICATION_HOME}/work` and `${APPLICATION_HOME}/temp`.
```
```suggestion
If a container start without anyuid enabled, applications can't write to `${APPLICATIOn_HOME}/logs`, `${APPLICATIOn_HOME}/work` and `${APPLICATIOn_HOME}/temp`.
```

Comment (nanux) [inline docs/docs/platforms/OPENSHIFT.md:35]:
```suggestion
If a container start without anyuid enabled, applications can't write to `${APPLICATION_HOME}/logs`, `${APPLICATION_HOME}/work` and `${APPLICATION_HOME}/temp`.
```
```suggestion
If a container start without anyuid enabled, applications can't write to `${APPLICATIOn_HOME}/logs`, `${APPLICATIOn_HOME}/work` and `${APPLICATIOn_HOME}/temp`.
```

File: docs/docs/platforms/OPENSHIFT.md [hunk 3]
```diff
@@ -26,11 +26,28 @@ It depends on the storage backend though.
...
+If you see in logs that the server fails to start with `permission denied` errors, you may want to declare these directories as runtime volumes. To do so, you need to declare additional volume mounts and additional volumes in values.yaml:
```
Comment (nanux) [inline docs/docs/platforms/OPENSHIFT.md:36]:
```suggestion
If you see in logs that the server fails to start with `permission denied` errors, you may want to declare these directories as runtime volumes. To do so, you need to declare additional volume mounts and additional volumes in `values.yaml`:
```
```suggestion
If you see in logs that the server fails to start with `permission denied` errors, you may want to declare these directories as runtime volumes. To do so, you need to declare additional volume mounts and additional volumes in values.yaml:
```

Comment (nanux) [inline docs/docs/platforms/OPENSHIFT.md:36]:
```suggestion
If you see in logs that the server fails to start with `permission denied` errors, you may want to declare these directories as runtime volumes. To do so, you need to declare additional volume mounts and additional volumes in `values.yaml`:
```
```suggestion
If you see in logs that the server fails to start with `permission denied` errors, you may want to declare these directories as runtime volumes. To do so, you need to declare additional volume mounts and additional volumes in values.yaml:
```

File: docs/docs/platforms/OPENSHIFT.md [hunk 4]
```diff
@@ -26,11 +26,28 @@ It depends on the storage backend though.
 
 ## Set no security context
```
Comment (nanux) [inline docs/docs/platforms/OPENSHIFT.md:27]:
```suggestion
## Disable security context
```

File: docs/docs/platforms/OPENSHIFT.md (hunk 1 above)
Comment (bianchi2) [reply docs/docs/platforms/OPENSHIFT.md]: thanks @nghazali Fixed

**Review by nanux — COMMENTED** [review]
Minor suggestions.

Thanks, @bianchi2 for the clarification!

**Review by nanux — APPROVED** [review]

**Review by nghazali — APPROVED** [review]

**Pull request merged**